website/data/docs-nav-data.json

```diff
@@ -483,6 +483,45 @@
             "title": "Overview",
             "path": "connect/gateways"
           },
+          {
```

Comment:
We probably need to include links to the new configuration entries in the side nav under Service Mesh > Configuration Entries, too.